.github/workflows/pr-akka-samples.yml

```diff
@@ -0,0 +1,94 @@
+
+  samples-shopping-cart-quickstart:
+    runs-on: ubuntu-22.04
+    # run only when `publish` workflow is successful or this workflow manually triggered
+    if: (github.event.workflow_run.conclusion == 'success' || github.event_name == 'workflow_dispatch')
+    steps:
+      - name: Checkout
+        # https://github.com/actions/checkout/releases
+        # v4.1.1
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11
+        with:
+          # we don't know what commit the last tag was it's safer to get entire repo so previousStableVersion resolves
+          fetch-depth: 0
+
+      - name: Cache Coursier cache
+        # https://github.com/coursier/cache-action/releases
+        # v6.4.5
+        uses: coursier/cache-action@1ff273bff02a8787bc9f1877d347948af647956d
+
+      - name: Set up JDK 21
+        # https://github.com/coursier/setup-action/releases
+        # v1.3.5
+        uses: coursier/setup-action@7bde40eee928896f074dbb76d22dd772eed5c65f
+        with:
+          jvm: temurin:1.21
+
+      - name: Determine SDK version
+        id: determine_sdk_version
+        run: |-
+          .github/determine-sdk-version.sh
+          SDK_VERSION="$(cat ~/akka-javasdk-version.txt)"
+          echo "SDK version: '${SDK_VERSION}'"
+          echo "sdk_version=${SDK_VERSION}" >> $GITHUB_OUTPUT
+
+      - name: Determine SDK version (manual triggered) # provisional until normal flow gets fixed)
+        if:  github.event_name == 'workflow_dispatch'
+        id: determine_sdk_version_manual_triggered
+        run: |-
+          .github/determine-sdk-version.sh
+          SDK_VERSION="$(cat ~/akka-javasdk-version.txt | sed 's/-.*//')" 
+          echo "SDK version: '${SDK_VERSION}'"
+          echo "sdk_version=${SDK_VERSION}" >> $GITHUB_OUTPUT
+
+      - name: Update SDK versions
+        env:
+          SDK_VERSION: ${{ steps.determine_sdk_version.outputs.sdk_version || steps.determine_sdk_version_manual_triggered.outputs.sdk_version }}
+        run: |-
+          echo "Updating samples to version ${SDK_VERSION}"
+          ./updateSdkVersions.sh all
+
+      - name: Checkout shopping-cart-quickstart
+        # https://github.com/actions/checkout/releases/tag/v4.2.2
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
+        with:
+          repository: akka-samples/shopping-cart-quickstart
+          path: akka-samples-repo-shopping-cart-quickstart
+          token: ${{ secrets.AKKA_SDK_RW_ACCESS_TOKEN }}
+          ref: main
+
+      - name: Open PR to shopping-cart-quickstart with changes
+        env:
+          COMMIT_URL: "https://github.com/${{ github.repository }}/commit/"
+          GH_TOKEN: ${{ secrets.AKKA_SAMPLES_RW_ACCESS_TOKEN }}
+        run: |
+          cd akka-samples-repo-shopping-cart-quickstart
+          git config user.name "github-actions[bot]"
+          git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
```

Comment:
not sure if this is the one we want to use? it was already used in this repo

Reply:
I was also surprised by this in a PR review, for the record It's a global github actions bot user id (https://api.github.com/users/github-actions%5Bbot%5D)